### PR TITLE
backtick error

### PR DESCRIPTION
**Pull request by youngjin17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 ## Installation
 Hanabot needs to run on an Ubuntu machine that is connected with a Baxter unit. Instructions for setting up Baxter can be found [here](http://sdk.rethinkrobotics.com/wiki/Baxter_Setup). 
-- Requirements: `python 3.7`, `pipenv`, 'python3.7-dev', 'ffmpeg'
```
Comment (youngjin17) [inline README.md]: what happened?

Comment (porouspaper) [reply README.md]: yay

Comment (noahsthompson) [reply README.md]: so good!

Comment (porouspaper) [reply README.md]: hello

File: README.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
-- Run 'sudo apt-get install libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0'
+- Requirements: `python 3.7`, `pipenv`, `python3.7-dev`, `ffmpeg`
+- Run `sudo apt-get install libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0`
 - In the directory of Hanabot, run `python3 -m pipenv install`
```
Comment (stephshum) [inline README.md]: test comment

File: README.md [hunk 3]
```diff
@@ -6,8 +6,8 @@
 
 ## Installation
 Hanabot needs to run on an Ubuntu machine that is connected with a Baxter unit. Instructions for setting up Baxter can be found [here](http://sdk.rethinkrobotics.com/wiki/Baxter_Setup). 
-- Requirements: `python 3.7`, `pipenv`, 'python3.7-dev', 'ffmpeg'
-- Run 'sudo apt-get install libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0'
+- Requirements: `python 3.7`, `pipenv`, `python3.7-dev`, `ffmpeg`
+- Run `sudo apt-get install libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0`
```
Comment (stephshum) [inline README.md]: test comment 2

File: README.md [hunk 4]
```diff
@@ -6,8 +6,8 @@
...
 - In the directory of Hanabot, run `python3 -m pipenv install`
 - To update, run `git pull`, then `python3 -m pipenv sync`
```
Comment (stephshum) [inline README.md]: test comment 3

**Pull request merged**